static/src/js/app/vuex/getters.js

```diff
@@ -5,37 +5,6 @@ export default {
     const knownTokens = tokens.filter(t => t.inVocabList).length;
     return knownTokens / totalTokens;
   },
-  weightedKnownVocab: (state) => {
-    const tokens = state.tokens.filter(t => t.word === t.word.toLowerCase());
-
-    // Create a set of unique tokens
-    const uniqueTokens = new Set();
-    for (let i = 0; i < tokens.length; i + 1) {
```

Comment:
@joshuagetega Is this the problematic line in question? Looks like there's a typo, since `i + 1` should be `i += 1` or `i++`. Without the assignment, this is just an infinite loop because `i` never increments.

Reply:
Ah, right. That's it. Thanks for catching it. I'm not sure how I did not catch that when I was testing the code after writing it. At any rate, I will rectify this by reintroducing the reverted changes, correcting the loop, testing the changes, and issuing a new PR with the corrected changes.